sdks/python/apache_beam/ml/inference/pytorch_inference_test.py

```diff
@@ -373,6 +373,40 @@ def test_invalid_input_type(self):
         # pylint: disable=expression-not-assigned
         pcoll | RunInference(model_handler)
 
+  def test_gpu_convert_to_cpu(self):
```

Comment:
why does this test fail over and work as CPU? Is it because the model is set up to be a CPU model, or is it because we are sure we are running the unit test on a device where the GPU fails?

Can you add some comments for that.

Reply:
The user specifies GPU using the `device='GPU'` arg. However, it detects that we're running the test on a machine without GPU, so it does automatic conversion to CPU.
```
        model_handler = PytorchModelHandlerTensor(
            state_dict_path=path,
            model_class=PytorchLinearRegression,
            model_params={
                'input_dim': 1, 'output_dim': 1
            },
            device='GPU')
``` 
Added clarifying comments.